components/curriculums/CurriculumInformation.js

```diff
@@ -0,0 +1,29 @@
+import { Typography, Alert, Card, CardContent } from "@mui/material";
+import { useOperation } from "react-openapi-client";
+import Loader from "../common/Loader";
+
+export default function CurriculumInformation({ curriculumId }) {
+  const {
+    loading,
+    data: curriculum,
+    error,
+  } = useOperation("getCurriculum", curriculumId);
```

Comment:
Just to be consistent with what we are doing everywhere else, you could use useFetch (cause it's a GET request), but it's not that important imo.

Reply:
Done!